lib/Install.js

```diff
@@ -117,6 +124,122 @@ function install(
   return 'SuccessfullyInstalled';
 }
 
+async function downloadFileNative(url, filePath) {
```

Comment:
Maybe we should add type annotations to all new functions in this file – for parameters and return types.

Reply:
Went ahead and did the last final touches: https://github.com/Janiczek/node-test-runner/pull/3